src/common/snippets/src/pass/insert_load_store.cpp

```diff
@@ -25,10 +25,13 @@ ngraph::snippets::pass::InsertLoad::InsertLoad(const size_t count) {
                 for (auto consumer : output.get_target_inputs()) {
                     // if a parameter is connected to a Load => we don't need another one
                     // if a parameter is connected to LoopBegin => there must be Load inside the Loop
+                    // if a parameter is connected to MatMul => we don't need Load (read/write is inside onednn kernel)
```

Comment:
`read/write is incapsulated into brgemm emitter` sounds better

Reply:
Ok, let's improve it